### PR TITLE
[xpu] support paddle compile with XFT

### PR DESCRIPTION
**Pull request by mayang002**
### PR types
New features

### PR changes
Others

### Describe
support Paddle compile with XPU-XFT

### CODE REVIEW

File: cmake/external/xpu.cmake [hunk 1]
```diff
@@ -116,7 +136,23 @@ if(WITH_XPU_BKCL)
...
   set(XPU_BKCL_INC_DIR "${THIRD_PARTY_PATH}/install/xpu/include")
   include_directories(${XPU_BKCL_INC_DIR})
+endif()
+
+if(WITH_XPU_XFT)
+  message(STATUS "Compile with XPU XFT!")
+  add_definitions(-DPADDLE_WITH_XPU_XFT)
+
+  set(XPU_XFT_LIB_DIR "${XPU_INSTALL_DIR}/lib")
+  set(XPU_XFT_LIB "${XPU_XFT_LIB_DIR}/${XPU_XFT_LIB_NAME}")
```
Comment (zhupengyang) [inline cmake/external/xpu.cmake]:
```suggestion
  set(XPU_XFT_LIB "${XPU_LIB_DIR}/${XPU_XFT_LIB_NAME}")
```

**Review by zhupengyang — DISMISSED** [review]
LGTM

**Review by zhupengyang — DISMISSED** [review]
LGTM

**Review by zyfncg — DISMISSED** [review]

**Review by risemeup1 — APPROVED** [review]
LGTM for setup.py

**Review by qili93 — APPROVED** [review]

**Review by zyfncg — APPROVED** [review]
LGTM for setup.py.in

**Review by zhupengyang — APPROVED** [review]

**Pull request merged**